descholar-ceo/AnnounceIT
# #170728121 User reset password

#### What does this PR ​ do​ ?
As a user, I should be able to reset my password

#### Description of Task to be completed?
* User clicks on reset password btn from the signin page
* User fills his email on reset password page
* Email check from the database
* User gets secret key on his/her email
* User enters the secret key in secret key field
* Check the validity of secret key
* User enters a password in passwords field
* User get redirected to his profile-page

#### How should this be manually tested?
* Clone this repository in case you don't have it using this command `git clone https://github.com/descholar-ceo/AnnounceIT.git`
* If you use [VSCode](https://code.visualstudio.com/) as code editor, install [_live-server_](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer), and click on _Go Live_

* Click on enter button from homepage, 

* And then you will be redirected in the browser, here you will need to go to the root of the url you have in url-bar, and enter like `http://127.0.0.1:5500/UI/html/signin.html`, you will be redirected to the signin page, or enter like this `http://127.0.0.1:5500/UI/html/signin.html`

* Or if you don't use [VSCode](https://code.visualstudio.com/) as text editor, after cloning the repo, you navigate in `AnnounceIT/UI/html/` and you will see different files, you choose `reset-password.html` and right click and open with your preferred web browser

#### Any background context you want to provide?
* It would be better if you use  [VSCode](https://code.visualstudio.com/)

#### What are the relevant pivotal tracker stories?
[170728121](https://www.pivotaltracker.com/n/projects/2429498#)

#### Screenshots (​ if​ appropriate)
![Screenshot from 2020-01-19 19-48-14](https://user-images.githubusercontent.com/56886064/72685786-1bbda280-3af6-11ea-8363-f2f81f86b093.png)
![Screenshot from 2020-01-19 19-48-44](https://user-images.githubusercontent.com/56886064/72685679-f2504700-3af4-11ea-9eb4-8bf1c73e6d1b.png)
![Screenshot from 2020-01-19 19-49-21](https://user-images.githubusercontent.com/56886064/72685680-f2e8dd80-3af4-11ea-9be0-6d3e000e80f9.png)


#### Questions:
No Question
